OTHER-DATABASES.md

```diff
@@ -0,0 +1,19 @@
+# Other databases
+
+Here is a list of some other Metamath databases that we know of
+but are not included in this repository:
+
+
+* @digama0 's [Dependent-Type Theory](https://github.com/digama0/dtt.mm/blob/master/dtt.mm) 
+* ~@sctfn 's older version of [New Foundations](https://github.com/sctfn/metamath-nf/blob/master/nf.mm) (this has since been moved into this repository)
+* @tirix 's [Q0 Logic](https://github.com/tirix/q0.mm)
+* @pbrosnan [Number Theory Game](https://github.com/pbrosnan/ntg) based on peano.mm, with additions.
+* @geohot 's [TwitchCoq](https://github.com/geohot/twitchcoq/blob/master/metamath/realtwoplustwo.mm) has a few interesting 2+2 proofs
+* @ishanpm 's [Linear Logic](https://github.com/ishanpm/metamathlinear/blob/master/linear.mm)
+* @jamesjer 's [Unlimited Register Machine and Hilbert Geometry](https://github.com/jamesjer/set.mm/blob/geometry/set.mm) is a fork of set.mm with very interesting work which has not been merged back
+* @sorear 's [A variation on Peano](https://github.com/sorear/metamath-stuff/blob/master/yapeano.mm) where expressions variables cannot fail to refer
```

Comment:
"expression variables" (no 's' at 'expression') ? By the way, what do you mean ?  If you mean the same thing as "fail to denote", then this is also the case in set.mm without class variables, since the logic used is classical (as opposed to free).

Reply:
To be honest I did not invest the time to understand Stefan's work. This seems to be an experiment, based on set.mm's predicate calculus, with this special axiom which caught my eye:
```
  ${
    $d x A $.
    $( This expresses the fact that in yapeano.mm, unlike in set.mm, expression
       variables cannot fail to refer. $)
    ax-ex $a |- E! x x = A $.
  $}
```
I did not give it further thoughts.

There is also a definition of the "unique individual such that" (`iota`) operator. It does not seem to be very much developed beyond definition and a few basic properties.

We shall maybe rather skip this entry altogether?

Reply:
What it means is that there is a sort of variables which represent natural numbers, rather than set.mm which has a sort for classes but no sort for set expressions. `peano.mm1` also has a sort for natural number expressions, because this is the most natural sort to give term constructors like `a + b` in peano arithmetic.

Reply:
Looking at the database though, it doesn't seem to actually have any peano arithmetic in it. Most of it is just FOL which looks copy-pasted from set.mm, except for the last section that defines a sort `exp` and an iota operator but doesn't leave the realm of pure FOL - there are no peano axioms, 0, + and so on.

Reply:
Ok, so it's a kind of FOL with terms.